apis/config/v1beta1/configuration_types.go

```diff
@@ -322,6 +322,7 @@ type Integrations struct {
 	//  - "kubeflow.org/xgboostjob"
 	//  - "pod"
 	//  - "deployment"
+	//  - "statefulset"
```

Comment:
In the issue, we have the requirement to `validate that it can only be enabled when pod integration is enabled`. Do we have this validation? 

Also I think it would be great to add a comment about it here.

Reply:
The comment is visible to the end-users, and so it probably makes sense to add it. The comments here play a role of user-facing guide. Similarly, docs may get out of sync, yet we try to maintain them. Maybe just:
`# requires enabling pod integration`